solidity/contracts/converter/types/fixed-rate-pool/FixedRatePoolConverter.sol

```diff
@@ -0,0 +1,315 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "../standard-pool/StandardPoolConverter.sol";
+
+/**
+ * @dev This contract is a specialized version of the converter, which is optimized for a
+ * liquidity pool that has 2 reserves with 50%/50% weights, and a conversion-rate of 1:1.
```

Comment:
"conversion rate of 1:1" isn't relevant anymore, can use "a constant predefined rate"

Reply:
Fixed (though it's not predefined, it is configurable by the owner of the converter).